pkg/cluster/deploybaseresources_additional.go

```diff
@@ -92,6 +92,20 @@ func (m *manager) clusterServicePrincipalRBAC() *arm.Resource {
 	)
 }
 
+func (m *manager) fpspStorageBlobContributorRBAC(storageAccountName string) (*arm.Resource, error) {
+	if !m.doc.OpenShiftCluster.UsesWorkloadIdentity() {
+		return nil, fmt.Errorf("fpspStorageBlobContributorRBAC called for a CSP cluster")
```

Comment:
what's CSP?  Don't use acronyms in case we need to debug.

Reply:
(I know what CSP is, just saying maybe call out Cluster service principal)

Reply:
Replaced the acronym everywhere.